site/docs/reference/Connectors/materialization-connectors/Firebolt.md

```diff
@@ -0,0 +1,83 @@
+* An S3 bucket in which to stage the external table
+* At least one Flow [collection](../../../concepts/collections.md)
+
+:::tip
+ If you haven't yet captured your data from its external source,
+ start at the beginning of the [guide to create a dataflow](../../../guides/create-dataflow.md).
+ You'll be referred back to this connector-specific documentation at the appropriate steps.
+:::
+
+## Configuration
+
+To use this connector, begin with data in one or more Flow collections.
+Use the below properties to configure a Firebolt materialization, which will direct Flow data to your desired Firebolt tables via an external table.
+
+### Properties
+
+#### Endpoint
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| `/aws_key_id` | AWS Key ID | AWS Key ID for accessing the S3 bucket. | string |  |
+| `/aws_region` | AWS Region | AWS Region the bucket is in. | string |  |
+| `/aws_secret_key` | AWS Secret Key | AWS Secret Key for accessing the S3 bucket. | string |  |
+| **`/database`** | Database | Name of the Firebolt database. | string | Required |
+| **`/engine_url`** | Engine URL | Engine URL of the Firebolt database, in the format: `<engine-name>.<organization>.<region>.app.firebolt.io`. | string | Required |
+| **`/password`** | Password | Firebolt password. | string | Required |
+| **`/s3_bucket`** | S3 Bucket | Name of S3 bucket where the intermediate files for external table will be stored. | string | Required |
+| `/s3_prefix` | S3 Prefix | A prefix for files stored in the bucket. | string |  |
+| **`/username`** | Username | Firebolt username. | string | Required |
+
+
+#### Bindings
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| **`/table`** | Table | Name of the Firebolt table to store materialized results in. The external table will be named after this table with an `_external` suffix. | string | Required |
+| **`/table_type`** | Table Type | Type of the Firebolt table to store materialized results in. See the [Firebolt docs](https://docs.firebolt.io/working-with-tables.html) for more details. | string | Required |
+
+### Sample
+
+```yaml
+materializations:
+  ${tenant}/${mat_name}:
+	  endpoint:
+        connector:
+          config:
+            database: my-db
+            engine_url: my-db-my-engine-name.my-organization.us-east-1.app.firebolt.io
+            password: secret
+            # For public S3 buckets, only the bucket name is required
+            s3_bucket: my-bucket
+            username: firebolt-user
+          # Path to the latest version of the connector, provided as a Docker image
+          image: ghcr.io/estuary/materialize-firebolt:dev
+	# If you have multiple collections you need to materialize, add a binding for each one
+    # to ensure complete data flow-through
+    bindings:
+      - resource:
+          table: table-name
+          table_type: FACT
```

Comment:
I actually think the code I wrote currently is case-sensitive lowercase and might not work with uppercase, might be worth changing that to be case-insensitive but just in case let's keep it lowercase for now:

```suggestion
          table_type: fact
```

Reply:
Got it, thanks, making that change. I assumed uppercase because Firebolt's docs refer to it in uppercase consistently, so I agreed it could be helpful to make it case-insensitive at some point.